df.announcements.xml

```diff
@@ -393,13 +393,6 @@
         <enum-item name='SATISFIED_MONARCH' since='v0.50.01'/>
         <enum-item name='MOUNTAINHOME' since='v0.50.01'/>
         <enum-item name='FOOD_WARNING' since='v0.50.01'/>
-
```

Comment:
please include the unknowns

`d_init_announcementst::flag` is a static array indexed by this enum, so having the enum have the correct number of items is essential to ensuring that this structure is properly aligned with this enum